src/components/core/DpAutocomplete.vue

```diff
@@ -110,7 +105,8 @@ export default {
     async fetchOptions (searchString) {
       this.isLoading = true
       try {
-        const response = await dpApi.get(Routing.generate(this.route, { ...this.additionalRouteParams, [this.queryParam]: searchString }))
+        const route = this.route(searchString)
```

Comment:
I think the prop "additionalRouteParams" can be removed now.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/91/commits/031f9e69bd8f952d615f8270755abb6dcefa813c